version2.en.md

```diff
@@ -1747,8 +1748,6 @@ Multiple asset objects containing img items may exist.
 
 ### Bid request: video with end-card
 
-Typically for rewarded video, where it can also be explicitly specified as `imp.video.ext.videotype = "rewarded"` or similar.
```

Comment:
Why are we deleting this?

Reply:
@yowcow Because we already have a bid request sample which include `imp.video.ext.videotype`. I think this description is a bit redundant at this time.

Reply:
The `imp.video.ext` is not a part of our spec, and it's probably a good idea to explicitly mention that it can include a specific signal for rewarded-video if demanded.

The new sample data contains `ext.rewarded_video = 1` which is already different from previous example `ext.videotype = "rewarded"`.

Reply:
Understood. Thank you for pointing it out.